administrator/components/com_media/Controller/Api.php

```diff
@@ -104,7 +105,20 @@ public function files()
 			switch ($method)
 			{
 				case 'get':
-					$data = $this->getModel()->getFiles($adapter, $path, $this->input->getWord('filter'));
+					$option = $this->input->getWord('opt');
```

Comment:
@dneukirchen do we need something like url => "path/to/url" or just "path/to/url" along with data in AJAX response ?